config/routes.rb

```diff
@@ -21,7 +21,7 @@
 
   root to: 'manage_io#index'
   patch '/projects/archive/:id' => 'projects#archive'
-  patch '/administrators/addAdmin/:id' => 'administrators#addAdmin'
+  patch '/administrators/addAdmin/:id' => 'administrators#add_admin'
```

Comment:
use snakecase in route path to be consistent with rest of the project
```suggestion
  patch '/administrators/add_admin/:id' => 'administrators#add_admin'
```